main.py

```diff
@@ -1,7 +1,31 @@
 import numpy as np
 import cv2
-from detector import Timestamp as T
+import sys
 
```

Comment:
We need it to be invoked from somewhere else, like another file or a package. Revert this change